CHANGELOG.md

```diff
@@ -68,6 +68,13 @@
 * [ENHANCEMENT] Flusher: Added `-flusher.exit-after-flush` option (defaults to true) to control whether Cortex should stop completely after Flusher has finished its work. #2877
 * [ENHANCEMENT] Added metrics `cortex_config_hash` and `cortex_runtime_config_hash` to expose hash of the currently active config file. #2874
 * [ENHANCEMENT] Logger: added JSON logging support, configured via the `-log.format=json` CLI flag or its respective YAML config option. #2386
+* [ENHANCEMENT] Support Snappy compression for gRPC. #2940
```

Comment:
I would rephrase this slightly as: Added new flags to configure compression used by gRPC (mention flags here). Valid values are "gzip", "snappy", "" (no compression, default). Previous flags (*.use-gzip-compression, enumerate) are now deprecated. WDYT?

Reply:
As we are deprecating things, we should move the deprecations to `CHANGE` as well.